--- before/posix.mak	2013-06-12 23:09:05.293332878 -0300
+++ after/posix.mak	2013-06-12 23:09:17.200340996 -0300
@@ -80,8 +80,8 @@
 
 # Set DRUNTIME name and full path
 ifeq (,$(findstring win,$(OS)))
-	DRUNTIME = $(DRUNTIME_PATH)/lib/libdruntime-$(OS)$(MODEL).a
-	DRUNTIMESO = $(DRUNTIME_PATH)/lib/libdruntime-$(OS)$(MODEL)so.a
+	DRUNTIME = /usr/lib/libdruntime-linuxdefault.a
+	DRUNTIMESO = /usr/lib/libdruntime-linuxdefaultso.a
 else
 	DRUNTIME = $(DRUNTIME_PATH)/lib/druntime.lib
 endif
